main.py

```diff
@@ -15,10 +15,128 @@
 import docker
 import teamcity
 import neo4j
-
+import argparse
 
 networks = ["the-bridge"]
 
+TestFlags = {
```

Comment:
Uppercase naming indicates class name... Feels strange to read ;-)
should be test_flags or perhaps testFlags (nice mix of these already...)

Reply:
Will correct the formatting.